website/docs/reference/resource-configs/athena-configs.md

```diff
@@ -0,0 +1,604 @@
+
+## AWS Lake Formation integration
+
+The following is how the adapter implements AWS Lake Formation tag management:
+
+- You can enable or disable lf-tags management via [config](#table-configuration) (disabled by default)
+- Once you enable the feature, lf-tags will be updated on every dbt run
+- First, all lf-tags for columns are removed to avoid inheritance issues
+- Then, all redundant lf-tags are removed from tables and actual tags from table configs are applied
+- Finally, lf-tags for columns are applied
+
+It's important to understand the following points:
+
+- dbt does not manage `lf-tags` for databases
+- dbt does not manage Lake Formation permissions
+
+That's why you should handle this by yourself manually or using an automation tool like terraform, AWS CDK, etc. You may find the following links useful to manage that:
+
+* [terraform aws_lakeformation_permissions](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lakeformation_permissions)
+* [terraform aws_lakeformation_resource_lf_tags](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lakeformation_resource_lf_tags)
+
+## Python models
+
+The adapter supports Python models using [`spark`](https://docs.aws.amazon.com/athena/latest/ug/notebooks-spark.html).
+
+### Setup
+
+- A Spark-enabled workgroup created in Athena
+- Spark execution role granted access to Athena, Glue and S3
+- The Spark workgroup is added to the `~/.dbt/profiles.yml` file and the profile to be used
+  is referenced in `dbt_project.yml`
+
+### Spark-specific table configuration
+
+- `timeout` (`default=43200`)
+  - Time out in seconds for each Python model execution. Defaults to 12 hours/43200 seconds.
+- `spark_encryption` (`default=false`)
+  - If this flag is set to true, encrypts data in transit between Spark nodes and also encrypts data at rest stored locally by Spark.
+- `spark_cross_account_catalog` (`default=false`)
+  - When using the Spark Athena workgroup, queries can only be made against catalogs located on the same AWS account by default. However, sometimes you want to query another catalog located on an external AWS account. Setting this additional Spark properties parameter to true will enable querying external catalogs. You can use the syntax `external_catalog_id/database.table` to access the external table on the external catalog (For example, `999999999999/mydatabase.cloudfront_logs` where 999999999999 is the external catalog ID)
+- `spark_requester_pays` (`default=false`)
+  - When an Amazon S3 bucket is configured as requester pays, the account of the user running the query is charged for data access and data transfer fees associated with the query.
+  - If this flag is set to true, requester pays S3 buckets are enabled in Athena for Spark.
+
```

Comment:
this is a bit dense. does it make sense to convert the above list into a table for easier scanning/reading?